ogame/__init__.py

```diff
@@ -26,7 +26,8 @@ def login(self):
         payload = {'kid': '',
                    'uni': self.server_url,
                    'login': self.username,
-                   'pass': self.password}
+                   'pass': self.password,
+                   'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64; rv:48.0) Gecko/20100101 Firefox/48.0'}                   
```

Comment:
This is not where the user-agent should be.
It should be put in the headers, not payload.